localstack/services/s3/s3_listener.py

```diff
@@ -649,6 +649,22 @@ def return_response(self, method, path, data, headers, response):
             except Exception:
                 pass
 
+            # Honor response header overrides
+            # https://docs.aws.amazon.com/AmazonS3/latest/API/RESTObjectGET.html
+            if method == 'GET':
+                query_map = urlparse.parse_qs(parsed.query, keep_blank_values=True)
+                allowed_overrides = {
```

Comment:
nitpick: could we move these to a top-level constant `ALLOWED_HEADER_OVERRIDES`? Not critical, but it would help us reduce the amount of "spaghetti code" we already have in here. :)

Reply:
No problem at all, I'll make the change.